tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -90,7 +123,9 @@ Create minimal VCH
     Property Should Contain         .runtime.power_state                 poweredOn
     Property Should Contain         .runtime.upgrade_status              Up to date
 
-    Wait For VCH Initialization  name=%{VCH-NAME}-api-test-minimal
+    ${vch_name}=    Set Variable    %{VCH-NAME}-api-test-minimal
```

Comment:
you only use this once, then don't use it in the teardown.  really not needed at all.

Reply:
You are correct.  This is an artifact from earlier testing.  I'll get rid of it.